### PR TITLE
[13.0][MIG] Migrated sale_stock_picking_blocking to v13

### PR DESCRIPTION
**Pull request by dreispt**
Supersedes #956

### CODE REVIEW

File: sale_stock_picking_blocking/i18n/sale_stock_picking_blocking_BACKUP_1781278.pot [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+msgstr ""
+
+#. module: sale_stock_picking_blocking
+#: model:ir.actions.act_window,name:sale_stock_picking_blocking.action_sale_stock_picking_blocking_reason
+#: model:ir.model,name:sale_stock_picking_blocking.model_sale_delivery_block_reason
+msgid "Sale Delivery Block Reason"
+msgstr ""
+
+#. module: sale_stock_picking_blocking
+<<<<<<< HEAD
```
Comment (rousseldenis) [inline sale_stock_picking_blocking/i18n/sale_stock_picking_blocking_BACKUP_1781278.pot]: @dreispt There are some rebasing errors here.

Comment (dreispt) [reply sale_stock_picking_blocking/i18n/sale_stock_picking_blocking_BACKUP_1781278.pot]: Hmm, those are backup files left by the original author. I guess they can be removed.

**Comment by dreispt** [comment]
@rousseldenis Files removed

**Comment by dreispt** [comment]
@rousseldenis This has already been migrated, and is on 13.0 branch 😞 . Closing.

**Pull request closed without merging**